k8s/helm/templates/chroma-namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: chroma
```

Comment:
@beggers, will it make sense to have `"{{ .Release.Namespace }}"` instead and let the user pick where things go in terms of namespace?